client/app/lib/dateTimeDefaults.js

```diff
@@ -1,19 +1,29 @@
-import moment from 'moment';
+import moment from 'lib/moment';
+
+const timeZone = document.head.querySelector("[name='server-context']")
+                     .getAttribute('data-time-zone');
 
 export const shortDateFormat = {
   year: 'numeric',
   month: 'numeric',
   day: 'numeric',
+  timeZone,
```

Comment:
Is this still necessary since both react-intl and moment supports timezone now ?